ReactNativeClient/lib/models/Setting.js

```diff
@@ -241,6 +241,7 @@ class Setting extends BaseModel {
 					let output = {};
 					output[Setting.THEME_LIGHT] = _('Light');
 					output[Setting.THEME_DARK] = _('Dark');
+					output[Setting.THEME_OLED_DARK] = _('OLED dark');
```

Comment:
As it is a mobile only theme, please make sure the option appears only on mobile

Reply:
Done

Reply:
Perferct, thanks!